client-sdks/client-js/src/resources/agent-builder.test.ts

```diff
@@ -130,3 +130,63 @@ describe('AgentBuilder.runs', () => {
     expect(params.get('resourceId')).toBe('test-resource-456');
   });
 });
+
+describe('AgentBuilder.transformWorkflowResult', () => {
```

Comment:
I don't think we need these tests, I think the workflow test is enough that we're passing through the correct type